Resources/Prototypes/_NF/Shipyard/kilderkin.yml

```diff
@@ -0,0 +1,29 @@
+- type: vessel
+  id: kilderkin
+  name: NC Kilderkin
+  description: "Spaceworthy bar/microbrewery with everything one needs to facilitate poor life choices: lots of booze, smokes, and lack of food."
+  price: 42900
+  category: Medium
+  group: Civilian
+  shuttlePath: /Maps/Shuttles/kilderkin.yml
+
+- type: gameMap
+  id: kilderkin
+  mapName: 'NC Kilderkin'
```

Comment:
Remove "NC" from the napName as NC should not appear in the IFF Tag.

Reply:
Done